quantus/metrics/axiomatic_metrics.py

```diff
@@ -67,7 +68,7 @@ def __init__(self, *args, **kwargs):
         self.disable_warnings = self.kwargs.get("disable_warnings", False)
         self.display_progressbar = self.kwargs.get("display_progressbar", False)
         self.output_func = self.kwargs.get("output_func", lambda x: x)
-        self.perturb_baseline = self.kwargs.get("perturb_baseline", "black")
+        self.perturb_baseline = self.kwargs.pop("perturb_baseline", "black")
```

Comment:
should we actually use pop here only for one kwargs key?

assuming we agreed on the changes in this PR https://github.com/understandable-machine-intelligence-lab/Quantus/issues/82